components/ui/LeftTimeline.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <!-- left timeline -->
+  <div
+    class="
+      mb-8
+      flex
+      md:justify-between
+      md:flex-row-reverse
+      flex-col
+      items-center
+      w-full
+      left-timeline
+    "
+  >
+    <div class="order-1 w-5/12"></div>
+    <div
+      class="
+        z-20
+        flex
+        items-center
+        order-1
+        bg-gray-800
+        shadow-xl
+        w-36
+        h-36
+        rounded-full
+      "
+    >
+      <img
```

Comment:
Can we have the image inside the card itself? Else its too small

Reply:
Dah edit, can you check again